lib/core/src/server/config/style.config.js

```diff
@@ -0,0 +1,47 @@
+import { logger } from '@storybook/node-logger';
+import autoprefixer from 'autoprefixer';
+import findUp from 'find-up';
+
+export default async function getStyleConfig(baseConfig) {
+  // null if file is not found
+  const customPostcssConfig = await findUp.sync('postcss.config.js', {
```

Comment:
it should be just `findUp(...)` without the sync

Reply:
🤦‍♂️  fixed